src/CattyTests/PlayerEngine/Sensors/Date/TimeHourSensorTest.swift

```diff
@@ -70,8 +70,8 @@ final class TimeHourSensorTest: XCTestCase {
     }
 
     func testConvertToStandardized() {
-        XCTAssertEqual(1, sensor.convertToStandardized(rawValue: 1))
-        XCTAssertEqual(10, sensor.convertToStandardized(rawValue: 10))
+        XCTAssertEqual(1, sensor.convertToStandardized(rawValue: 1, landscapeMode: false))
+        XCTAssertEqual(10, sensor.convertToStandardized(rawValue: 10, landscapeMode: false))
```

Comment:
Please also add a test for `convertToStandardized` when called with `landscapeMode` `true`.